charts/prometheus-process-exporter/templates/daemonset.yaml

```diff
@@ -44,14 +44,21 @@ spec:
               containerPort: {{ .Values.service.targetPort }}
               protocol: TCP
           livenessProbe:
+            failureThreshold: {{ .Values.livenessProbe.failureThreshold }}
             httpGet:
               path: /
               port: {{ .Values.service.port }}
-              initialDelaySeconds: 20
+            initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
+            successThreshold: {{ .Values.livenessProbe.successThreshold }}
```

Comment:
You can safely rewrite L47, L51-L53 shorter as the below :)

```
{{ if .Values.livensesProbe }}{{ toYaml .Values.livenessProbe | indent 12 }}{{ end }}
```

The same thing can be said for `readinessProebe` as well

Reply:
Done :)